decentraland/unity-renderer
# fix: mesh and material leaks on avatar lods

## What does this PR change?

Material and meshes were being accessed using the `material` and `mesh` property instead of `sharedMaterial` and `sharedMesh`. This caused small material/mesh leaks on avatars.

## How to test the changes?

The easiest way to test is to add/remove 100 bots from Unity and check the memory details on the profiler. The material/mesh count as shown by the profiler should go back to the original values.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

-----
Wondermine testing: https://play.decentraland.zone/?renderer-branch=fix/mesh-and-material-leaks-on-avatar-lods&ENABLE_AVATAR_LODS&MAX_VISIBLE_PEERS=200&position=-28%2C55

Stadium testing: https://play.decentraland.zone/?renderer-branch=fix/mesh-and-material-leaks-on-avatar-lods&ENABLE_AVATAR_LODS&MAX_VISIBLE_PEERS=200&position=91,151&NETWORK=ropsten
